src/modules/lunchs/models/Order.js

```diff
@@ -42,46 +42,72 @@ module.exports = (knex = data) => {
       })
   }
 
-  const groupOrders = (data, order = 'asc') => {
-    const orders = {}
+  const retriveProductsFromOrder = async (ids = []) => {
```

Comment:
```suggestion
  const retrieveProductsFromOrder = async (ids = []) => {
```
faltou um 'e' no retrieve